react-elements/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom';
+
+class Hello extends React.Component {
+  render() {
+    return React.createElement(
+      'h1',
+      null,
+      'Hello, React!'
+    );
+  }
+}
+
+console.log(Hello);
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(React.createElement(Hello, { toWhat: 'World' }, null));
```

Comment:
Good job with this but it is a bit overboard for this exercise. You just need to create an element then render it. Creating a Component is getting a bit ahead.
```JAVASCRIPT
const h1 = React.createElement('h1', null, 'Hello, React!');

console.log(h1);

const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(h1);
```